Sources/ClaimSet.swift

```diff
@@ -1,3 +1,6 @@
+import Foundation
+import CryptoSwift
```

Comment:
I understand that `Foundation` is required, but I don't think `CryptoSwift` should be. There is no use of CryptoSwift throughout this file.